src/components/ViolenceTestModal/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Row, Button, Modal } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import "./index.scss";
+
+const ViolenceTestModal = () => {
+  const [show, setShow] = useState(false);
```

Comment:
isShown, setIsShown

try to start boolean variables with words like is, has, etc to make it more clear
"show" is a verb, a noun, etc. so especially its ambiguous

Reply:
it could be "something to be shown". it could be a function that does showing. calling it isShown makes it 100% clear immediately

Reply:
done